lm_eval/base.py

```diff
@@ -15,8 +13,11 @@
 import torch
 import torch.nn.functional as F
 
-from lm_eval import metrics
-from lm_eval.metrics import mean, weighted_perplexity, weighted_mean, bits_per_byte
+from lm_eval.metrics import (
+    mean,
+    weighted_perplexity,
+    bits_per_byte,
+)
 from lm_eval import utils
```

Comment:
```suggestion
from lm_eval import utils, metrics
```

Need metrics.rouge, metrics.sari